include/wx/xml/xml.h

```diff
@@ -238,8 +248,8 @@ class WXDLLIMPEXP_XML wxXmlDocument : public wxObject
 
     // Parses .xml file and loads data. Returns TRUE on success, FALSE
     // otherwise.
-    bool Load(const wxString& filename, int flags = wxXMLDOC_NONE);
-    bool Load(wxInputStream& stream, int flags = wxXMLDOC_NONE);
+    bool Load(const wxString& filename, int flags = wxXMLDOC_NONE, wxXmlParseError* err_details = nullptr);
```

Comment:
Very minor, but we use `camelCase` rather than `snake_case` convention, so this should rather be `errDetails` (or just `err` if you'd prefer).